examples/fields.go

```diff
@@ -6,16 +6,31 @@ import (
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+	"os"
 )
 
+func checkEnvironmentVariableExists(host, user, password string) {
+	if host == "" || user == "" || password == "" {
+		fmt.Println("ERROR: Missing one of these environment variables: SDK_TARGET_HOST, SDK_TARGET_USER, SDK_TARGET_PASSWORD")
+		fmt.Println("See README for usage")
+		os.Exit(1)
+	}
+}
+
 func main() {
+	host := os.Getenv("SDK_TARGET_HOST")
+	user := os.Getenv("SDK_TARGET_USER")
+	password := os.Getenv("SDK_TARGET_PASSWORD")
+
+	checkEnvironmentVariableExists(host, user, password)
```

Comment:
Is there a reason to put this in a separate function?  I thought it could stay inline like I showed in my previous comment.